bazel/curl.BUILD

```diff
@@ -0,0 +1,32 @@
+load("@io_opencensus_cpp//opencensus:curl.bzl", "CURL_COPTS")
+
+package(features = ['no_copts_tokenization'])
+
+config_setting(
+    name = "windows",
+    values = {"cpu": "x64_windows"},
+    visibility = [ "//visibility:private" ],
+)
+
+config_setting(
+    name = "osx",
+    values = {"cpu": "darwin"},
+    visibility = [ "//visibility:private" ],
+)
+
+cc_library(
+    name = "curl",
+    srcs = glob([
+        "lib/**/*.c",
+    ]),
+    hdrs = glob([
+        "include/curl/*.h",
+        "lib/**/*.h",
+    ]),
+    includes = ["include/", "lib/"],
+    copts = CURL_COPTS + [
+        "-DOS=\"os\"",
```

Comment:
"-DOS=os",

Reply:
It should be `\"os\"`.
If I change this to `os`, there is an error:
```
external/com_github_curl/lib/version.c:302:3: error: use of undeclared identifier 'os'
  OS, /* as found by configure or set by hand at build-time */
  ^
<command line>:162:12: note: expanded from here
#define OS os
```

Reply:
Ooops! I'm wrong. Please change it back.